package.json

```diff
@@ -12,18 +12,21 @@
 		"*.{js,jsx,ts,tsx,css,md,json}": "prettier --write",
 		"package.json": "sort-package-json"
 	},
-	"dependencies": {
+	"devDependencies": {
 		"@changesets/cli": "^2.24.1",
 		"@guardian/prettier": "^2.0.0",
 		"@manypkg/get-packages": "^1.1.3",
 		"@nrwl/cli": "14.4.3",
+		"@nrwl/devkit": "^14.5.4",
 		"@nrwl/eslint-plugin-nx": "14.4.3",
-		"@nrwl/jest": "14.4.3",
+		"@nrwl/jest": "14.5.4",
 		"@nrwl/js": "14.4.3",
 		"@nrwl/linter": "14.4.3",
 		"@nrwl/node": "^14.4.3",
 		"@nrwl/nx-cloud": "latest",
 		"@nrwl/workspace": "^14.4.3",
+		"@swc-node/register": "^1.5.1",
+		"@swc/core": "^1.2.224",
```

Comment:
NX uses these instead of `ts-node` when it compiles internal builders and executors on the fly, for speed.